client/app/bundles/course/video/submission/store.js

```diff
@@ -1,6 +1,38 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import thunkMiddleware from 'redux-thunk';
-import rootReducer, { createInitialState } from './reducers';
+import { Map as makeImmutableMap } from 'immutable';
+import { persistReducer, persistStore } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import rootReducer, { createInitialState, persistTransforms } from './reducers';
+
+const stateReconciler = (inboundState, _, reducedState) => {
```

Comment:
Would be good to write some specs for this. It might be hard to decipher the intent of this if someone doesn't read your PR message.